bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableOptionsFactory.java

```diff
@@ -454,6 +454,8 @@ private static void setChannelOptions(
     // This information is in addition to bigtable-client-core version, and jdk version.
     StringBuilder agentBuilder = new StringBuilder();
     agentBuilder.append("hbase-").append(VersionInfo.getVersion());
+    agentBuilder.append(", ");
+    agentBuilder.append("bigtable-hbase-").append(BigtableHBaseVersion.getVersion());
```

Comment:
My bad, I should have ported https://github.com/googleapis/java-bigtable-hbase/pull/3033/files. Mind if I do that first to reduce the noise on this PR?